codewindtechpreview.md

```diff
@@ -58,13 +57,12 @@ Projects must be located in `<user>/microclimate-workspace`.
 
 #### Installing the Eclipse plug-in
 1. Download and install the latest Eclipse IDE for Java EE Developers or use an existing installation. The earliest supported version of the Eclipse IDE is Version 4.8 (Photon).
-2. Open the [Eclipse IDE](https://marketplace.eclipse.org/content/codewind) and go to **Help**>**Eclipse Marketplace**.
-3. Search for `Codewind`.
-4. Click **Install**.
-5. Finish the wizard and accept licenses as prompted.
-6. When the installation is complete, restart Eclipse.
-7. Open the Codewind Explorer view. Go to **Window**>**Show View**>**Other** and type `Codewind` in the filter field.
-8. Then, select **Codewind Explorer** and click **Open**.
+2. Obtain the plug-in in one of the following ways: 
+    - Open the [Eclipse IDE](https://marketplace.eclipse.org/content/codewind) and go to **Help**>**Eclipse Marketplace**. Then, search for `Codewind`. Click **Install** and finish the wizard and accept licenses as prompted.
+   - You can also use the [Marketplace Drag to Install button](https://marketplace.eclipse.org/content/codewind#group-external-install-button).
```

Comment:
We want to use the drag-to-install to add an install button to the page that the user can drag into their running eclipse to install.  I believe the markdown to add the button looks like this
```
[![Drag to your running Eclipse* workspace. *Requires Eclipse Marketplace Client](https://marketplace.eclipse.org/sites/all/themes/solstice/public/images/marketplace/btn-install.png)](http://marketplace.eclipse.org/marketplace-client-intro?mpc_install=4638524 "Drag to your running Eclipse* workspace. *Requires Eclipse Marketplace Client")
```